static/css/base.css

```
@@ -59,8 +59,25 @@ body{
   position: absolute;
   border: 2px solid black;
   border-radius: 100px;
-  left: 20%;
-  top: 100px;
+  left: 53%;
+  top: 180px;
   z-index: 1;
   display: none;
 }
+#player-chip {
```

No šī te visa pilnīgi pietiktu ar:
```suggestion
#player-chip {
  top: 500px;
}
```